src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/basic/GT_MetaTileEntity_CropHarvestor.java

```diff
@@ -270,9 +270,9 @@ && getBaseMetaTileEntity().isAllowedToWork()
                                     ItemStack[] aHarvest = tCrop.harvest_automated(true);
                                     if (aHarvest != null && aHarvest.length > 0) {
                                         for (ItemStack aStack : aHarvest) {
-                                            if (aStack.stackSize > 0) {
+                                            if (aStack != null && aStack.stackSize > 0) {
                                                 if (this.mTier * 5 > MathUtils.randInt(1, 100)) {
-                                                    aStack.stackSize += Math.floor(tCrop.getGain() / 10);
+                                                    aStack.stackSize += Math.floor((float) tCrop.getGain() / 10);
```

Comment:
The floor is not necessary here, integer division rounds towards zero

Reply:
true